backend/functions/lib/adapters/gateways/cases.azure.sql.gateway.ts

```diff
@@ -64,15 +69,20 @@ class Chapter11ApiGateway implements Chapter11GatewayInterface {
       });
     }
 
-    const queryResult: QueryResults = await executeQuery(context, query, input);
+    const queryResult: QueryResults = await executeQuery(
+      context,
+      context.config.acmsDbConfig,
```

Comment:
I think we either just past the context or just what is needed from context here.

A third option here could be just to pass in config here. If the reason for passing in context is for logging, we could let the error message propagate and log it in this class.

Another option which would probably take the most work though is to create a repository object to query anything related to cases. Also refactor log object as an object instead of a static methods. We should talk about this one if interested in this approach.

Reply:
The reason we pass `context.config.acmsDbConfig` is because that is how we communicate to the `executeQuery` function which database to run the query against. In other places we pass `context.config.dxtrDbConfig`. I would prefer to remove chapter 11 stuff entirely and then we don't need to keep track of two different sql configs.